MediaRecorder.html

```diff
@@ -442,6 +443,38 @@
               <em>Return type:</em> <code>boolean</code>
             </div>
           </dd>
+
+          <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
+          <dd>
+            When <code>getSupportedTypes()</code> method is invoked, the UA MUST
+            queue a task, using the DOM manipulation task source, that runs the
```

Comment:
I have no clue, I think I siphoned this phrasing off from someplace else describing how to express correctly Promise-based methods, but is possible that I frankensteneised it in flight. How would you phrase it?